.travis.yml

```diff
@@ -13,4 +13,4 @@ script:
   - make coverage
   - bash <(curl -s https://codecov.io/bash) -f .coverage/combined.cover.out
   - make integration
-  - make master-integration
+#  - make master-integration
```

Comment:
will follow up quickly with another pr to re-enable this step

Reply:
info: we may consider removing this build step entirely if we feel it continues to be problematic. Its been more noise than signal for me lately as well.

Reply:
We need to think about alternatives re: not breaking API. As the test from master looks like the only thing that does it (even if poorly).